### PR TITLE
🆕🎨 Design updates IV

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Restyle and fixes for the token table
- Click on token -> links to SELL token selected on the SWAP container
- Changed ACTIONS: Replace BUY/SELL with Etherscan icon (VIEW CONTRACT) + APPROVE status. On click of the token name itself it will go to the SWAP page, selecting the token as a SELL token. This to simplify the token table UI.
- Mobile optimize (for now horizontal scroll)

Screen Shot 2022-11-18 at 13 13 23
Screen Shot 2022-11-18 at 13 13 36

Screen Shot 2022-11-18 at 13 14 00

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , great changes!!
Some issues: 
1. The page content is truncated when screen width is in [960-1090]px
reduced
2. Mobile: lists dropdown is truncated
image
3. Mobile: Search area is mostly hidden on Favourites list
image
4. It would be great not to truncate a placeholder in the search field:
image

5. It would be great to expand a message area when no search results
image

Thanks!

**Comment by fairlighteth** [comment]
@elena-zh thanks for reviewing. Addressed all points except:
1) Let's address this separately 
4) Also let's address this separately. We could show a shorter text for mobile.

**Review by shoom3301 — APPROVED** [review]
LGTM!

**Comment by elena-zh** [comment]
Hey @fairlighteth , 
I have created #1444 issue for the 1, 4 cases
cases 5-6 LGTM!

As for the rest ones: the table does not look good now on a reduced screen: 
image

https://user-images.githubusercontent.com/70885163/203036690-c962b47d-fc02-4abb-98fb-906a9d9528f2.mp4


Could you please fix it? 
Thanks!

**Comment by fairlighteth** [comment]
@elena-zh Pushed some updates. For resolutions around 960 <-> 1024px there are still some issues, which I like to address later.

**Comment by elena-zh** [comment]
Hey @fairlighteth , great job!
One nitpick is that 'hovered' effect looks like truncated line on reduced screens, and when the table is scrolled. 
hover is truncated

Should I open a separate issue for this?

File: src/custom/components/Tokens/TokensTableRow.tsx [hunk 1]
```diff
@@ -189,19 +185,12 @@ const DataRow = ({
...
-          <TableButton color={theme.green1}>Buy</TableButton>
-        </Link>
-      </Cell>
-
-      <Cell>
-        <Link to={tradeLink(tokenData, OrderKind.SELL)}>
-          <TableButton color={theme.red1}>Sell</TableButton>
-        </Link>
+        <ExtLink href={getBlockExplorerUrl(chainId, tokenData.address, 'token')}>
+          <TableButton>Contract</TableButton>
```
Comment (anxolin) [inline src/custom/components/Tokens/TokensTableRow.tsx:189]: I find this a bit strange

I would actually expect the opposite, click on the token would take me to Etherscan, and the link would read as "Trade".

not super strong opinion though

**Review by anxolin — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
Approved with #1496 issue

**Comment by fairlighteth** [comment]
@elena-zh Can you check the latest? I fixed issue #1496

**Comment by elena-zh** [comment]
Hey @fairlighteth , #1496 issue looks great to me here. I have also created #1498 issue to address low-priority issues related to the Contract icon

**Pull request merged**